pqcli/roster.py

```diff
@@ -29,6 +30,8 @@ def save(self) -> None:
         tmp_path.parent.mkdir(parents=True, exist_ok=True)
         tmp_path.write_bytes(pickle.dumps(self.players))
         if self.path.exists():
+            if os.path.exists(old_path):
+                os.remove(old_path)
```

Comment:
Would this work instead:

```
if old_path.exists():
    old_path.unlink()
```
? If so, we don't need the `os` import

Reply:
I give it a try

Reply:
Yep, it does the trick

Reply:
Are you willing to update your patch or would you like me to push the fix? :)

Reply:
i did it in second PR - better way via separate branch
please accept it